content/en/blog/_posts/Introducing-Kueue.md

```diff
@@ -0,0 +1,208 @@
+also currently no good way to control at the job level which jobs should get which resources first, and no way to express order or fair sharing. The 
+current ResourceQuota model is not a good fit for our needs as quotas are enforced on resource creation, and so there is no queueing of requests. The 
+intent of ResourceQuotas is to provide a builtin reliability mechanism with policies needed by admins to protect clusters from failing over.
+
+In the Kubernetes ecosystem, there are several solutions for job scheduling; we found that these alternatives have one or more of the following problems:
+- They replace existing stable components of Kubernetes, like kube-scheduler. This is problematic not only from an operational point of view, but 
+  also the duplication in the job APIs causes fragmentation of the ecosystem.
+- They don’t integrate with autoscaling, or 
+- They lack support for resource flexibility. 
+
+## Kueue overview {#overview}
+With Kueue we decided to take a different approach to job queueing on Kubernetes that is anchored around the following aspects: 
+- Not duplicating existing functionalities already offered by established Kubernetes components for pod scheduling, autoscaling and job
+  lifecycle management. 
+- When key features are missing, we add them to existing components. For example, we invested in the Job API to cover more use cases like 
+  [IndexedJob](/blog/2021/04/19/introducing-indexed-jobs) and [fixed long standing issues related to pod 
+  tracking](/docs/concepts/workloads/controllers/job/#job-tracking-with-finalizers). While this path takes longer to 
+  land features, we believe it is the more sustainable long term solution.
+- Ensuring compatibility with cloud environments where compute resources are elastic and heterogeneous.
+
+For this approach to be feasible, Kueue needs knobs to influence the behavior of those established components so it can effectively manage 
+when and where to start a job. We added those knobs to the 'Job' API in the form of two features:
+- [Suspend field](/docs/concepts/workloads/controllers/job/#suspending-a-job), which allows Kueue to signal to the job controller 
+  when to start or stop a job.
+- [Mutable scheduling directives](/docs/concepts/workloads/controllers/job/#mutable-scheduling-directives), which allows Kueue to 
+- ```
+  update a Job’s 'nodeSelector' before starting the job. This way, Kueue can control pod placement while still delegating to kube-scheduler the actual
+  pod-to-node scheduling.
+
+Note that any custom job API can be managed by Kueue if that API offers the above two capabilities.
+
+### Resource model
+Kueue defines new APIs to address the requirements mentioned at the beginning of this post. The three main APIs are:
+
+- 'ResourceFlavor': a cluster-scoped API to define resource flavor available for consumption, like a GPU model. At its core, a ResourceFlavor is 
```

Comment:
```suggestion
- ResourceFlavor: a cluster-scoped API to define resource flavor available for consumption, like a GPU model. At its core, a ResourceFlavor is 
```

We generally don't use ticks for resource objects.

Reply:
removed here and other places.